package.json

```diff
@@ -54,7 +54,6 @@
     "ember-collection": "1.0.0-alpha.5",
     "ember-data": "2.7.0",
     "ember-data-factory-guy": "^2.6.1",
-    "ember-disable-prototype-extensions": "^1.0.0",
```

Comment:
Ember-SHARE was using prototype extensions and the majority of this PR was directly pulled from that repo. Prototype extensions worked fine in preprints and registries, so I removed this addon.  

However, I have rewritten the sections of this PR that are using prototype extensions and restored the addon.